framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -1175,11 +1176,14 @@ public void addAnnotation(AnnotationMirror a) {
          *
          * @param params the parameter types, excluding the receiver
```

Comment:
Should we add something like `will be captured by method` to the documentation for these parameters to highlight that only a unique reference should be passed in?

Reply:
Yes, it's an important note for the callers. I've updated the javadoc for all related setter methods.